src/_scss/pages/covid19/recipient/recipientContainer.scss

```diff
@@ -4,6 +4,11 @@
   .count-tabs {
     &.recipient__tabs-container {
       width: 100%;
+      .usa-dt-tab-list {
+        z-index: 15;
+        overflow-x: visible;
```

Comment:
<img width="445" alt="Screen Shot 2021-06-30 at 12 43 49 PM" src="https://user-images.githubusercontent.com/7108785/123999757-d8c95500-d9a0-11eb-9236-1b6c8ea7f7f4.png">
This change is causing the tabs to overflow in mobile - was there another issue you were trying to fix?

Reply:
Ohhh I I see it is for the tooltip in the tab. I brought this up as an anti-pattern and accessibility issue on another ticket. Might be a larger discussion. I'll see if I can find that Slack thread!